distmaker/dists/common.sh

```diff
@@ -211,7 +211,7 @@ function dm_generate_bower() {
 function dm_generate_vendor() {
   local repo="$1"
   pushd "$repo"
-    ${DM_COMPOSER:-composer} install
+    ${DM_COMPOSER:-composer} install --ignore-platform-reqs
```

Comment:
@totten Think we should at the very least wrap this only doing this if < 5.3.27 php